pom.xml

```diff
@@ -66,140 +66,144 @@
         
         <jakarta.jakartaee-bom.version>8.0.0</jakarta.jakartaee-bom.version>
         <glassfish-jaxb.version>2.3.9</glassfish-jaxb.version>
         
         <atomikos.version>6.0.0</atomikos.version>
-        <narayana.version>5.12.7.Final</narayana.version>
-        <jboss-transaction-spi.version>7.6.1.Final</jboss-transaction-spi.version>
-        <jboss-logging.version>3.2.1.Final</jboss-logging.version>
+        <narayana.version>5.13.1.Final</narayana.version>
+        <jboss-transaction-spi.version>8.0.0.Final</jboss-transaction-spi.version>
+        <jboss-logging.version>3.6.1.Final</jboss-logging.version>
         <seata.version>2.2.0</seata.version>
         
-        <netty.version>4.1.112.Final</netty.version>
-        <bouncycastle.version>1.78.1</bouncycastle.version>
+        <netty.version>4.1.117.Final</netty.version>
+        <bouncycastle.version>1.80</bouncycastle.version>
         
-        <curator.version>5.7.0</curator.version>
-        <zookeeper.version>3.9.2</zookeeper.version>
-        <audience-annotations.version>0.12.0</audience-annotations.version>
-        <jetcd.version>0.7.7</jetcd.version>
-        <vertx.version>4.5.1</vertx.version>
+        <curator.version>5.7.1</curator.version>
+        <zookeeper.version>3.9.3</zookeeper.version>
+        <audience-annotations.version>0.15.0</audience-annotations.version>
+        <jetcd.version>0.8.4</jetcd.version>
+        <vertx.version>4.5.11</vertx.version>
         
-        <grpc.version>1.65.1</grpc.version>
-        <protobuf.version>3.21.12</protobuf.version>
+        <grpc.version>1.69.1</grpc.version>
+        <protobuf.version>4.29.3</protobuf.version>
         <okhttp.version>4.12.0</okhttp.version>
         
         <elasticjob.version>3.0.4</elasticjob.version>
         
-        <slf4j.version>1.7.36</slf4j.version>
-        <logback.version>1.2.13</logback.version>
-        <commons-logging.version>1.2</commons-logging.version>
+        <slf4j.version>2.0.16</slf4j.version>
+        <logback.version>1.5.16</logback.version>
+        <commons-logging.version>1.3.4</commons-logging.version>
         
         <lombok.version>1.18.36</lombok.version>
-        <immutables.version>2.9.3</immutables.version>
+        <immutables.version>2.10.1</immutables.version>
         
-        <postgresql.version>42.7.2</postgresql.version>
-        <mysql-connector-java.version>8.3.0</mysql-connector-java.version>
-        <mssql.version>6.1.7.jre8-preview</mssql.version>
-        <h2.version>2.2.224</h2.version>
-        <opengauss.version>3.1.0-og</opengauss.version>
-        <mariadb-java-client.version>2.4.2</mariadb-java-client.version>
-        <clickhouse-jdbc.version>0.6.3</clickhouse-jdbc.version>
+        <postgresql.version>42.7.5</postgresql.version>
+        <mysql-connector-java.version>9.1.0</mysql-connector-java.version>
+        <mssql.version>12.9.0.jre8-preview</mssql.version>
+        <h2.version>2.3.232</h2.version>
```

Comment:
- The new version of h2database has dropped support for JDK8, which is not reasonable.